db/re/item_combo_db.txt

```diff
@@ -349,6 +379,19 @@
 2990:2991,{ bonus bAllStats,5; bonus5 bAutoSpellWhenHit,"SC_MAELSTROM",1,100,BF_MAGIC,0; }
 2995:15110,{ bonus2 bSkillAtk,"NC_AXEBOOMERANG",100; }
 2997:2999,{ bonus2 bAddClass,Class_All,10; bonus bMatkRate,10; }
+32242:18971,{ bonus2 bFixedCastrate,"RK_STORMBLAST",-100; }
+32242:18973,{ bonus2 bSkillAtk,"NC_FLAMELAUNCHER",70; bonus2 bSkillAtk,"NC_COLDSLOWER",70; }
```

Comment:
on iRO, Old Detachments Ring has a set bonus with Old Mitra (18972) (50% hp bonus)
32242:18972,{ bonus bMaxHPrate,50; }
can't confirm if its iRO exclusive or not

Reply:
It seem I miss that haha.

Reply:
done